x/stakeibc/keeper/msg_server_restore_interchain_account_test.go

```diff
@@ -0,0 +1,66 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	_ "github.com/stretchr/testify/suite"
+
+	stakeibc "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+type RestoreInterchainAccountTestCase struct {
+	validMsg stakeibc.MsgRestoreInterchainAccount
+}
+
+func (s *KeeperTestSuite) SetupRestoreInterchainAccount() RestoreInterchainAccountTestCase {
+	s.CreateTransferChannel(HostChainId)
+
+	hostZone := stakeibc.HostZone{
+		ChainId:      HostChainId,
+		HostDenom:    Atom,
+		Bech32Prefix: GaiaPrefix,
```

Comment:
I think these can be removed, only chainId and connection should be necessary

Reply:
done

Reply:
done